<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生选修课程系统 SECS - 登录</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_User">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>104</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>帐号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Pwd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>132</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Type">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>类型：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_user">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>104</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_pwd">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>132</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Login">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>登 录</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>160</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string> 学生</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 管理员</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_Welcome">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>欢迎来到 学生选修课程系统 SECS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
